kalite/coachreports/templates/coachreports/report_view.html

```diff
@@ -0,0 +1,245 @@
+{% extends "coachreports/base.html" %}
+
+{% load i18n %}
+{% load kalite_staticfiles %}
+{% load my_filters %}
+
+{% block coachreports_active %}active{% endblock coachreports_active %}
+{% block title %}{% trans "Progress by topic" %} {{ block.super }}{% endblock title %}
+
+{% block headcss %}{{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/coachreports/report_view.css' %}" />
+        <link rel="stylesheet" type="text/css" href="{% static 'css/jquery-ui/plugins/ui.dynatree.css' %}">
+        <link rel="stylesheet" type="text/css" href="{% static 'css/jquery-ui/plugins/ui.dynatree.defaultoff.css' %}">
+
+{% endblock headcss %}
+
+
+
+{% block headjs %}{{ block.super }}
+    <script type="text/javascript" src="{% static 'js/coachreports/jquery.cookie.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/jquery.dynatree.min.js' %}"></script>
+    <script type="text/javascript" src="{% static 'js/jquery.dynatree.extensions.js' %}"></script>
+
+    <script>
```

Comment:
New and large chunks of JS should be:

- Not inline in the document, but included as external JS files
- Using Backone.js and Handlebars rather than ad-hoc JS